addon/package.json

```diff
@@ -86,9 +63,37 @@
     "access": "public",
     "registry": "https://registry.npmjs.org"
   },
+  "files": [
+    "declarations",
+    "dist",
+    "src",
```

Comment:
@NullVoxPopuli why would we need src to be published?

Reply:
so go-to-definition can go to source instead of declarations (requires declaration maps, iirc)